plugins/linear-genome-view/src/LinearGenomeView/index.ts

```diff
@@ -221,13 +222,42 @@ export function stateModelFactory(pluginManager: PluginManager) {
         )
       },
 
-      bpToPx({ refName, coord }: { refName: string; coord: number }) {
+      idxInDisplayedRegion(refName: string | undefined) {
```

Comment:
Is this function needed still?

Reply:
No, actually it was still marking it as implemented because I added a test, but I can remove that method.